packages/atomic/cypress/integration/search-box-selectors.ts

```diff
@@ -1,7 +1,14 @@
+import {AriaLiveSelectors} from './aria-live-selectors';
+
+export const searchBoxComponent = 'atomic-search-box';
+
 export const SearchBoxSelectors = {
-  component: 'atomic-search-box',
-  inputBox: 'input',
-  querySuggestionList: 'ul',
+  shadow: () => cy.get(searchBoxComponent).shadow(),
+  inputBox: () => SearchBoxSelectors.shadow().find('[part="input"]'),
+  submitButton: () =>
+    SearchBoxSelectors.shadow().find('[part="submit-button"]'),
+  querySuggestions: () => SearchBoxSelectors.shadow().find('[data-query]'),
```

Comment:
Unfortunately, `data-query` and some CSS classes seem to be the only differences between a suggestion and a button (such as Clear recent queries) that's inside the suggestions box.